src/main/resources/application.yml

```diff
@@ -0,0 +1,24 @@
+spring:
+  datasource:
+    initialization-mode: always
```

Comment:
you use Liquibase, why do you need this property enabled?

Reply:
[done]